examples/DigitalRain/DigitalRain.ino

```diff
@@ -29,10 +28,24 @@ KEYMAPS(
 
 // *INDENT-ON*
 
+KALEIDOSCOPE_INIT_PLUGINS(
+
+  // LEDControl provides support for other LED modes
+  LEDControl,
+
+  // We start with the LED effect that turns off all the LEDs.
+  LEDOff,
+
+  // Support for DigitalRain
+   LEDDigitalRainEffect
+
+  );
+
+  
 void setup() {
   Kaleidoscope.setup();
 
-  Kaleidoscope.use(&LEDControl, &LEDDigitalRainEffect, &LEDOff);
+  LEDDigitalRainEffect.DROP_TICKS = 22; // Make the rain fall faster
```

Comment:
I think no need to change the configuration here in this file -- having it in the readme is enough, and this example file can be a bare minimum -- so remove this line (and the whitespace above) please.

Reply:
Done.